readme.txt

```diff
@@ -8,58 +8,80 @@ License: GPLv2 or later
+
+## AMP Ecosystem
+
+It is possible today to assemble great looking user-first sites powered by the AMP plugin by picking and choosing themes and plugins from a growing AMP-compatible ecosystem. In this context, the AMP plugin acts as an orchestrator of the overall AMP content creation and publishing process; it serves as a validator and enforcer making it easier to not only to get to AMP experiences, but to stay in them with confidence.
+
+Many popular theme and plugin developers have taken efforts to support The Official AMP Plugin. If you are using a theme like Astra, Newspack or plugins like Yoast, WP Forms — they will work out of the box! You can see the growing list of tested themes and plugins [here](https://amp-wp.org/ecosystem/).
+
+## AMP Development
+
+Although there is a growing ecosystem of AMP compatible WordPress components, still there are some ways to go before 100% AMP compatibility in the ecosystem. If you are a developer, or you have the resources to pursue development projects, you may want, in some cases, develop custom functionality (i.e. as a plugin, or in the theme space) to serve your specific needs. The Official AMP Plugin can be of great help to you by providing powerful and effective developer tools that shed light into the AMP development process as it is done in WordPress, including mechanisms for detailing the root causes of all validation issues, and the contextual space to understand them properly, and dealing with them during the process of achieving full AMP compatibility. Read more about [Developer Tools](https://amp-wp.org/documentation/getting-started/developer-tools/).
 
 == Frequently Asked Questions ==
 
-Please see the [FAQs on amp-wp.org](https://amp-wp.org/documentation/frequently-asked-questions/). Don't see an answer to your question? Please [search the support forum](https://wordpress.org/support/plugin/amp/) to see if someone has asked your question. Otherwise, please [open a new support topic](https://wordpress.org/support/plugin/amp/#new-post).
+Please see the [FAQs on amp-wp.org](https://amp-wp.org/documentation/frequently-asked-questions/). Don&#39;t see an answer to your question? Please [search the support forum](https://wordpress.org/support/plugin/amp/) to see if someone has asked your question. Otherwise, please [open a new support topic](https://wordpress.org/support/plugin/amp/#new-post).
 
 == Installation ==
 
 1. Upload the folder to the `/wp-content/plugins/` directory.
-2. Activate the plugin through the 'Plugins' menu in WordPress.
+2. Activate the plugin through the &#39;Plugins&#39; menu in WordPress.
 3. If you currently use older versions of the plugin in `Reader` mode, it is strongly encouraged to migrate to `Transitional` or `Standard` mode. Depending on your theme/plugins, some development work may be required.
```

Comment:
```suggestion
3. If you currently use older versions of the plugin in `Reader` mode, it is strongly encouraged to pick a Reader theme instead of using the legacy Reader templates. You may also want to migrate to `Transitional` or `Standard` mode; depending on your theme/plugins, some development work may be required.
```